src/main/java/command/FindCommand.java

```diff
@@ -0,0 +1,30 @@
+package command;
+
+import java.util.ArrayList;
+
+import duke.Storage;
+import duke.Ui;
+import duke.UiForGUI;
+import task.Task;
+import task.TaskList;
+
+/**
+ * The FindCommand class is a type of Command that is used to find the matching tasks.
+ */
+public class FindCommand extends Command {
+    public FindCommand(String command, String[] tokenizedCommand) {
+        super(command, tokenizedCommand);
+    }
+
+    /**
+     * Executes the find command and displays the list of tasks that matches the keyword.
+     * @param tasks TaskList that stores the current list of tasks.
```

Comment:
Do you need to put `@return` in a new line? I noticed this in the other files as well. You could consider getting IntelliJ to [autoformat](https://stackoverflow.com/questions/17879475/how-enable-auto-format-code-for-intellij-idea) your files. 😄 
<img width="355" alt="Screenshot 2022-02-03 at 8 38 45 PM" src="https://user-images.githubusercontent.com/61443625/152344331-8a6dab6c-b359-4e53-8f60-1361858906f8.png">